packages/account/src/Sections/Verification/ProofOfIdentity/proof-of-identity.jsx

```diff
@@ -18,6 +19,20 @@ const ProofOfIdentity = ({
     routeBackInApp,
     should_allow_authentication,
 }) => {
+    // next useEffect implements seo requirements
+    React.useEffect(() => {
+        const description_content = 'Submit your proof of identity documents to verify your account and start trading';
```

Comment:
Just a suggestion: you can move `description_content` and `title_content` out of the component

Reply:
Leaving it as it is after our conversation in slack :)